tests/policies/test_replay_buffer_wrapper.py

```diff
@@ -0,0 +1,94 @@
+"""Tests ExplorationWrapper."""
+
+import os.path as osp
+from typing import Type
+
+import numpy as np
+import pytest
+import stable_baselines3 as sb3
+from stable_baselines3.common import buffers, off_policy_algorithm, policies
+from stable_baselines3.common.policies import BasePolicy
+from stable_baselines3.common.save_util import load_from_pkl
+
+from imitation.policies.replay_buffer_wrapper import ReplayBufferRewardWrapper
+from imitation.util import util
+
+
+def zero_reward_fn(
+    state: np.ndarray,
+    action: np.ndarray,
+    next_state: np.ndarray,
+    done: np.ndarray,
+):
+    del action, next_state, done
+    return np.zeros(state.shape[0], dtype=np.float32)
+
+
+def make_wrapper_n_algo(
```

Comment:
what does the `_n_` refer to here?

Reply:
Oh, I meant `and`. Maybe it's better to name it as `make_algo_with_wrapped_buffer`